corehq/apps/repeaters/models.py

```diff
@@ -313,31 +315,13 @@ def handle_response(self, result, repeat_record):
         """
         if isinstance(result, Exception):
             repeat_record.handle_exception(result)
-            self.handle_exception(result, repeat_record)
+            self.generator.handle_exception(result, repeat_record)
```

Comment:
:) well if it was, I've certainly also used that pattern before in other code then, because it seems very familiar...